chart/values.yaml

```diff
@@ -121,7 +121,7 @@ images:
     pullPolicy: IfNotPresent
   gitSync:
     repository: registry.k8s.io/git-sync/git-sync
-    tag: v3.6.9
+    tag: v4.0.0
```

Comment:
I think we should use the `v4.1.0` version of git-sync by default.
This will fix issues when you use NFS for your DAG's folder. You can see more info in the following link: https://github.com/kubernetes/git-sync/issues/827

Reply:
Thank you for the review. I changed the default version.